src/main/java/edu/ie3/datamodel/models/input/thermal/CylindricalStorageInput.java

```diff
@@ -147,17 +163,19 @@ public String toString() {
   public static class CylindricalStorageInputCopyBuilder
       extends ThermalStorageInputCopyBuilder<CylindricalStorageInputCopyBuilder> {
 
-    private ComparableQuantity<Volume> storageVolumeLvl;
-    private ComparableQuantity<Temperature> inletTemp;
-    private ComparableQuantity<Temperature> returnTemp;
-    private ComparableQuantity<SpecificHeatCapacity> c;
+    ComparableQuantity<Volume> storageVolumeLvl;
+    ComparableQuantity<Temperature> inletTemp;
+    ComparableQuantity<Temperature> returnTemp;
+    ComparableQuantity<SpecificHeatCapacity> c;
+    ComparableQuantity<Power> pThermalMax;
```

Comment:
Why did you make this `package private`?

Reply:
It's not, it was private.

Reply:
Yes, it was private, but now it is package private.

Reply:
To use it at `DomesticHotWaterStorageInputCopyBuilder`. But it could stay private if we make the `CylindricalStorageInputCopyBuilder` protected, right?

Reply:
Since you removed most of the code in the `DomesticHotWaterStorageInputCopyBuilder`, these fields can even stay private.

Reply:
I guess this is now obsolete with the change to `AbstractThermalStorageInput`.